### DISCUSSION TITLE
Add an intro skipper feature

### DISCUSSION THREAD

**Discussion by kris3713** [DonutWare/Fladder#164] [Ideas]
### Summary

Just as the title describes.

### Basic Example

The intro skipper plugin should use the popular intro skipper plugin by jumoog to skip intros in TV show episodes and skip credits as well.

### Drawbacks

None. I can't think of any so far.

### Unresolved questions

None.

**Comment by PartyDonut** [comment]
This is already supported. 
For Fladder > 0.4.4 make sure you're using the latest Jellyfin because that version uses mediasegments (intro skipper uses those now).
